src/components/CCPDashboard/PatientOverview.tsx

```diff
@@ -176,6 +177,7 @@ export const PatientOverview = (props: PatientOverviewProps) => {
           eventId={eventId}
           ccpId={ccpId}
           patientId={patientId}
+          numUpdates={numUpdates}
```

Comment:
Since you're passing everything to `PatientInfoTableWithFilters`, you should be able to do just do this `{...props}` to make it cleaner.